CAIPs/caip-220.md

```diff
@@ -0,0 +1,97 @@
+---
+caip: CAIP-X
+title: Block Addressing
+author: Juan Caballero (@bumblefudge), Sebastian Posth <sebastian@posth.com>, and Antonio Antonioni (@ntn-x2)
+discussions-to: https://github.com/chainAgnostic/CAIPs/pull/220
+status: Draft
+type: Standard
+created: 2023-03-20
+updated: 2023-03-20
+requires: 2
+replaces: <CAIP number(s); if multiple, format as `[1,2]` array>
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the CAIP.-->
+Universal syntax for addressing specific bundles of verifiable transactions
+(i.e. "blocks") in blockchains, directed acyclical graphs, or other
+cryptographical data systems.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+While namespace-specific syntaxes will always exist and be preferable for single-namespace 
+
+## Motivation
+<!--The motivation is critical for CAIP. It should clearly explain why the state of the art is inadequate to address the problem that the CAIP solves. CAIP submissions without sufficient motivation may be rejected outright.-->
+While block explorers for newer systems tend to inherit their user experience
+and syntax from prior art, there is much divergence in both their public-facing
+routes (see [CAIP-200][]) and in the properties queriable per block. This
+specification defines a minimal set of commonly-used properties to serve as a
+translation layer across those syntaxes, with per-namespace profiles needed to
+further constraint the values and add namespace-specific properties that may be
+important to cross-namespace use cases.
+
+## Specification
+<!--The technical specification should describe the standard in detail. The specification should be detailed enough to allow competing, interoperable implementations. -->
+Blocks are addressed as follows:
+
+```
+block_address:        chain_id + ":block/" + account_address + ["." + property]?
+chain_id:             [-a-z0-9]{3,8}:[-_a-zA-Z0-9]{1,32} (See [CAIP-2][])
```

Comment:
Do we need to repeat the definition of `chain_id` in all documents that use it? Wouldn't just linking to it be enough? I see that otherwise managing all occurrences might be hell.